README.md

```diff
@@ -0,0 +1,44 @@
+# CPSC 304 Project - Me Collect
+### Project Overview
+Me Collect is a robust collection management system where users can record data about their own collectibles and look for other items that they can add to their collection. 
+
+### Features
+* Users can sign-in, view their personal collection of collectibles and wishlist, and browse through the collectibles database
+* Users can filter collectibles by various attributes (e.g. brand, price, and year)
+* Sellers can create new collectible objects
+
+### Team
+* Daniel Anatolie - 
+* Alex Ford - 
+* Shreeshaa Goyal -
+* Frances Sin - 16888142
+
+### Documents
+* [Milestone 1b - Project Proposal]()
+* [Milestone 2 - ERD and Schema]()
+* [Milestone 3 - FD and Normalization]()
+* [Milestone 4 - Formal Specification]()
+* [Milestone 5 - Complete Project]()
+* [Milestone 6 - Demo]()
+
+### Requirements
+@todo
+```
+@todo
+```
+
+### Usage
+@todo
```

Comment:
Add: 
Start the client & server with:
yarn           # builds needed node_modules
yarn dev    # starts client & server 
Visit localhost3000

Reply:
Done